src/main/java/org/usfirst/frc/team319/robot/subsystems/BBArm.java

```diff
@@ -24,15 +24,15 @@
  */
 public class BBArm extends Subsystem implements IPositionControlledSubsystem {
 
-  public LeaderBobTalonSRX bbaLead = new LeaderBobTalonSRX(11, new BobTalonSRX(12));
-  public LeaderBobTalonSRX bbCollectTalon = new LeaderBobTalonSRX(13);
+  public LeaderBobTalonSRX bbaLead = new LeaderBobTalonSRX(10, new BobTalonSRX(8));
+  public LeaderBobTalonSRX bbCollectTalon = new LeaderBobTalonSRX(9);
```

Comment:
Inconsistent naming between `bbaLead` and `bbCollectTalon`  I suggest `bbaCollector`